[2024-11-14T21:11:02.934+0900] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-14T21:11:02.945+0900] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.first_task scheduled__2024-11-14T11:34:00+00:00 [queued]>
[2024-11-14T21:11:02.951+0900] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.first_task scheduled__2024-11-14T11:34:00+00:00 [queued]>
[2024-11-14T21:11:02.951+0900] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-11-14T21:11:02.966+0900] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2024-11-14 11:34:00+00:00
[2024-11-14T21:11:02.968+0900] {standard_task_runner.py:72} INFO - Started process 2378224 to run task
[2024-11-14T21:11:02.972+0900] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'first_task', 'scheduled__2024-11-14T11:34:00+00:00', '--job-id', '293', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpapi_gplf']
[2024-11-14T21:11:02.973+0900] {standard_task_runner.py:105} INFO - Job 293: Subtask first_task
[2024-11-14T21:11:03.008+0900] {task_command.py:467} INFO - Running <TaskInstance: first_dag.first_task scheduled__2024-11-14T11:34:00+00:00 [running]> on host MN
[2024-11-14T21:11:03.076+0900] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-14T11:34:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-14T11:34:00+00:00'
[2024-11-14T21:11:03.077+0900] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-14T21:11:03.092+0900] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-14T21:11:03.092+0900] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the fist task!']
[2024-11-14T21:11:03.097+0900] {subprocess.py:99} INFO - Output:
[2024-11-14T21:11:03.098+0900] {subprocess.py:106} INFO - hello world, this is the fist task!
[2024-11-14T21:11:03.099+0900] {subprocess.py:110} INFO - Command exited with return code 0
[2024-11-14T21:11:03.117+0900] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-14T21:11:03.117+0900] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=first_task, run_id=scheduled__2024-11-14T11:34:00+00:00, execution_date=20241114T113400, start_date=20241114T121102, end_date=20241114T121103
[2024-11-14T21:11:03.144+0900] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-14T21:11:03.156+0900] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-14T21:11:03.156+0900] {local_task_job_runner.py:245} INFO - ::endgroup::
